src/averageAges.js

```diff
@@ -34,7 +49,25 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let womenWithChildren = [];
```

Comment:
the same mistake as in the previous function, no need to use an empty array, just assign filtered array.